x-pack/test/security_solution_cypress/cypress/e2e/investigations/alerts/detection_page_filters.cy.ts

```diff
@@ -222,7 +221,7 @@ describe.skip(`Detections : Page Filters`, { tags: ['@ess', '@serverless'] }, ()
   });
 
   context('with data modificiation', () => {
```

Comment:
I know I'm being annoying here, but I feel like this title could be a bit more descriptive?
Also `modificiation` => `modification`

Reply:
You are good @PhilippeOberti . This is how a review should look like. Please carry on with what you are doing.

Reply:
this is done.

Reply:
Let me see how I can make more descriptive.